Core/include/Acts/Seeding/detail/CylindricalSpacePointGrid.ipp

```diff
@@ -130,9 +130,19 @@ Acts::CylindricalSpacePointGridCreator::createGrid(
     }
   }
 
+  std::vector<AxisScalar> rValues{};
+  rValues.reserve(std::max(2ul, config.rBinEdges.size()));
+  if (config.rBinEdges.empty()) {
+    rValues = {config.rMin, config.rMax};
+  } else {
+    rValues.insert(rValues.end(), config.rBinEdges.begin(),
+                   config.rBinEdges.end());
+  }
+
   Axis<AxisType::Variable, AxisBoundaryType::Bound> zAxis(std::move(zValues));
+  Axis<AxisType::Variable, AxisBoundaryType::Bound> rAxis(std::move(rValues));
```

Comment:
Not important, but I just want to advertise that `Axis` can now to template deduction, like:

```suggestion
  Axis rAxis(std::move(rValues));
```

without having to specify the template arguments explicitly.